src/Approximations/overapproximate.jl

```diff
@@ -1635,3 +1635,66 @@ function overapproximate(X::LazySet{N}, ::Type{<:Zonotope},
     end
     return Zonotope(c, G)
 end
+
+"""
+    overapproximate(r::Rectification{N, <:Zonotope{N}}, ::Type{<:AbstractZonotope}) where {N}
+
+Overapproximation of the rectification of a zonotopic set.
+
+### Input
+
+- `r` -- Lazy Rectification of a zonotope
+- `Zonotope` -- type for dispatch
+
+### Output
+
+The zonotope overapproximation of the set obtained by rectifying `Z`.
```

Comment:
```suggestion
A zonotope overapproximation of the set obtained by rectifying `Z`.
```
because such overapproximation is not unique :)